proposals/language/README.md

```diff
@@ -0,0 +1,251 @@
+* [Unicode CLDR: Picking the Right Language Identifier](https://cldr.unicode.org/index/cldr-spec/picking-the-right-language-code)
+* [W3C: Language Tags and Local Identifiers for the World Wide Web](https://www.w3.org/TR/ltli/)
+* [Unicode: Language Tag Equivalences](https://cldr.unicode.org/index/cldr-spec/language-tag-equivalences)
+* [Common list of Locales](https://gist.github.com/typpo/b2b828a35e683b9bf8db91b5404f1bd1)
+* [Apple: Choosing localization regions and Scripts](https://developer.apple.com/documentation/xcode/choosing-localization-regions-and-scripts)
+
+## Details
+
+### What would use it?
+
+This addition to USD is designed to be generic over several other schema types that might benefit from it.
+
+The primary use case is the
+current [Text proposal from Autodesk](https://github.com/PixarAnimationStudios/OpenUSD-proposals/tree/main/proposals/text)
+, where text is a really good pairing for language specification.
+
+Hypothetically, in the future, we could see it also being useful for other use cases like:
+
+- User facing assistive metadata
+- Texture
+- Geometry
+
+We do not expect these to support languages right away, but we believe this is a good future looking feature that
+would allow for the use of USD in specific multi-language pipelines.
+
+For this proposal, we do not require that other schemas explicitly adopt language support. We suggest that this is
+something that can be adopted in runtimes over time to support in conjunction with other schemas.
+
+### Language Encoding
+
+To maximize compatibility with other systems, we recommend using `BCP-47` derived locales. For use as a `purpose`,
+this would require the use of `_` as a delimiter , as opposed to the standard `-`.
+
+e.g. Instead of `en-CA`, we use `en_CA`
+
+This brings it closer to the derived `Unicode Common Locale Data Repository (CLDR)` standard. This is commonly used
+by many operating systems, programming languages and corporations. If you are on a POSIX system, this also has
+significant overlap with the POSIX locale standards ([ISO/IEC 15897](https://www.iso.org/standard/50707.html)).
+
+An example list of languages is provided in the relevant links section above.
+
+### Unspecified Language Fallback
+
+In the event that a language is not specified, it is recommended to specify a fallback behaviour.
+
+Our recommendation is:
+
+1. If your attribute or prim is missing a language, check the parent hierarchy for an inherited value
+2. If no language is specified, and if your runtime can infer a language, it is free to do so but does not have to.
+3. If you cannot or chose not to infer a language, assume the users current locale.
+
+This matches the behaviour of common assistive technologies like screen readers.
+
+### Default Metadata
+
+Most content will prescribe to one primary language, which tends to be the region of the content creator.
+To facilitate this, we encourage but do not require content authors to specify a language.
```

Comment:
I agree with "encourage but not require." 👍🏼 There's a lot of HTML content in the wild marked as `en-us` when it's clearly not.